courses/free-courses-en.md

```diff
@@ -163,6 +164,12 @@
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790)
 * [Shell Scripting Tutorial for Beginners](https://www.youtube.com/playlist?list=PLS1QulWo1RIYmaxcEqw5JhK3b-6rgdWO_) (ProgrammingKnowledge)
 
+### Blockchain
+
+* [Dapp University](https://www.youtube.com/c/DappUniversity)
+* [Moralis Web3](https://www.youtube.com/c/MoralisWeb3)
+* [Gajesh S Naik](https://www.youtube.com/c/GajeshSNaik/featured)
+* [EatTheBlocks](https://www.youtube.com/c/EatTheBlocks)
```

Comment:
:information_source: For YouTube, we prefer "playlist" links and the name of the producer/creator.

Fix also linter faults